### PR TITLE
Fix list-groups in text browsers

### PR DESCRIPTION
**Pull request by mmagorsc**
Closes: https://bugs.gentoo.org/711286

Signed-off-by: Max Magorsch <arzano@gentoo.org>

### CODE REVIEW

**Comment by ulm** [comment]
It fails validation:
```
$ make tidy 
Failed on eclass-reference/index.html:
line 66 column 1 - Warning: missing </div> before <li>
line 67 column 1 - Warning: inserting implicit <ul>
line 67 column 1 - Warning: missing </ul> before </div>
make: *** [Makefile:86: tidy] Error 1
```

**Comment by mmagorsc** [comment]
> It fails validation:
> 
> ```
> $ make tidy 
> Failed on eclass-reference/index.html:
> line 66 column 1 - Warning: missing </div> before <li>
> line 67 column 1 - Warning: inserting implicit <ul>
> line 67 column 1 - Warning: missing </ul> before </div>
> make: *** [Makefile:86: tidy] Error 1
> ```

Oh, I missed to replace div by ul. Should be fixed now.

File: devmanual.css [hunk 1]
```diff
@@ -73,4 +73,17 @@ pre span.Statement { color: darkred; }
     margin-top: 3px;
 }
 
+/*
+ * .fix-links can be used to fix listgroups in text browsers
+ * see https://bugs.gentoo.org/show_bug.cgi?id=711286
+ */
+
+.fix-links > li {
+    padding:0rem;
```
Comment (ulm) [inline devmanual.css]:
```suggestion
    padding: 0rem;
```

**Comment by ulm** [comment]
Merged to master.

**Pull request closed without merging**